adapters/celestia/src/da_service.rs

```diff
@@ -106,6 +107,10 @@ fn default_max_response_size() -> u32 {
     1024 * 1024 * 100 // 100 MB
 }
 
+fn default_request_timeout_ms() -> u64 {
+    10_000
+}
```

Comment:
let's make it `const`

Reply:
Why?

I made it this way, as ground work for https://github.com/Sovereign-Labs/sovereign-sdk/issues/478

Reply:
Ok I thought it will be a hardcoded value.